apollo-router/src/plugins/telemetry/config_new/attributes.rs

```diff
@@ -37,7 +38,11 @@ where
     E: JsonSchema,
 {
     fn schema_name() -> String {
-        "extendable_attribute".to_string()
+        format!(
```

Comment:
The attributes were conflicting in the json schema, so make sure they have unique names.